### ISSUE TITLE
[bug]: unable to sweep funds > non-mandatory-script-verify-flag (Invalid Schnorr signature)

### ISSUE THREAD

**Issue by stakoholic** [lightningnetwork/lnd#9119]
Hello, after my last channel was co-op closed few days ago from remote, i want to sweep all my funds and i get this message

FAIL: [lncli] rpc error: code = Unknown desc = unable to broadcast sweep transaction: unmatched backend error: -26: non-mandatory-script-verify-flag (Invalid Schnorr signature)

the bug was mentioned here  https://github.com/raspiblitz/raspiblitz/issues/3364 by @openoms  and @guggero  but cant find any further infos to this issue.

i have a  raspberry pi 4 /4gb and raspiblitz v1.10 with Linux raspberrypi 6.1.21-v8+ 
bitcoind v25.0.0 + lnd 0.17.3-beta

**Comment by bitromortac** [comment]
Could you join https://lightning.engineering/slack.html and post a message there (and/or send me a dm), it could be easier to help interactively?

**Comment by stakoholic** [comment]
alright, thx

**Comment by dstadulis** [comment]
@stakoholic 
- What date was the lnd instance / wallet first created?  or if you're not certain, the earliest possible creation date
- Was it created on the raspberry pi?

**Comment by stakoholic** [comment]
@dstadulis yes it was created on the raspi and the creation date was at least 2 years ago

**Comment by dstadulis** [comment]
@stakoholic was the creation date before March 2022?

**Comment by stakoholic** [comment]
looked up for the first tx and it was August 2020
